website/src/components/blog/BlogPost.tsx

```diff
@@ -46,9 +46,17 @@ export const BlogPost: React.FunctionComponent<Props> = ({
                         post.frontmatter.title
                     )}
                 </h1>
-                <p className="blog-post__byline mb-0">
-                    {post.frontmatter.author} on {post.frontmatter.publishDate}
-                </p>
+                {post.frontmatter.author && post.frontmatter.publishDate && (
+                    <p className="blog-post__byline mb-0">
+                        {post.frontmatter.authorUrl ?
+                            <a href={post.frontmatter.authorUrl} target="_blank" rel="nofollow noreferrer">
+                                {post.frontmatter.author}
+                            </a> :
+                            <span>{post.frontmatter.author}</span>
```

Comment:
There are a few cases where blog posts have more than one author. Now that the link is introduced, those blogposts group both authors into one hyperlink, sometimes to a site representing just one of the two authors. I don't think it's within the scope of this ticket, but it may be useful to support multiple author/authorUrl pairings in the future.

/blog/real-weakest-link-in-software-supply-chain-security/
<img width="600" alt="Screen Shot 2022-05-04 at 9 17 52 AM" src="https://user-images.githubusercontent.com/60713139/166701019-6244b2eb-137e-4c39-be58-967894b0155f.png">

/blog/remote-software-development-made-easier/
<img width="600" alt="Screen Shot 2022-05-04 at 9 19 20 AM" src="https://user-images.githubusercontent.com/60713139/166701301-6624681a-1e39-4f49-bc80-d4a8aef168b3.png">

Reply:
That sounds good!